### PR TITLE
add org.freedesktop.fldigi

### PR DESCRIPTION
**Pull request by lu9dce**
Fldigi is a modem program for most of the digital modes used by radio amateurs today: CW, PSK, MFSK, RTTY, Hell, DominoEX, Olivia, and Throb are all supported. It can help calibrate a sound card to a time signal and do frequency measurement tests

### CODE REVIEW

File: fldigi.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+app-id: org.freedesktop.fldigi
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: fldigi
+finish-args:
+  - --share=ipc
+  - --socket=x11
```
Comment (bilelmoussaoui) [inline fldigi.yml:8]: If it runs under wayland, switch this one to `fallback-x11`

File: fldigi.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  - name: portaudio
+    sources:
+      - type: git
+        url: https://github.com/PortAudio/portaudio.git
+
+  - name: hamlib
+    sources:
+      - type: git
+        url: https://github.com/Hamlib/Hamlib.git
+
```
Comment (bilelmoussaoui) [inline fldigi.yml:32]: Should be ideally tarballs otherwise tagged release/specific commit.

File: fldigi.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+sdk: org.freedesktop.Sdk
+command: fldigi
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
```
Comment (bilelmoussaoui) [inline fldigi.yml:13]: What does it needs home access for?

File: fldigi.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+app-id: org.freedesktop.fldigi
```
Comment (bilelmoussaoui) [inline fldigi.yml:1]: The file should be renamed to `org.freedesktop.fldigi.yml`

File: fldigi.yml (hunk 1 above)
Comment (lu9dce) [reply fldigi.yml:8]: ok tks

File: fldigi.yml (hunk 4 above)
Comment (lu9dce) [reply fldigi.yml:1]: ok

File: fldigi.yml (hunk 3 above)
Comment (lu9dce) [reply fldigi.yml:13]: in the user's home folder it keeps a logbook of the contacts made
user must be able to send text from file
you think it is not necessary can be removed
it's my first flatpak, have me passience

**Pull request closed without merging**